test/manifolds/projective_space.jl

```diff
@@ -324,4 +324,13 @@ include("../utils.jl")
             end
         end
     end
+    @testset "Weingarten & Hessian" begin
+        M = PositiveNumbers()
```

Comment:
A test for `PositiveNumbers` in `projective_space.jl`?

Reply:
so late in the alphabet with testing, I was probably already a bit tired. ;) Moved it.